libs/utils/lana.js

```diff
@@ -111,6 +111,7 @@
     debug: false,
     log,
     options: mergeOptions(w.lana && w.lana.options),
+    loaded: true,
```

Comment:
Does it need this? IMO it should be enough to tell if `window.lana` exists, if it does you can assume it has loaded

Reply:
This is not true. Try it. `window.lana` is defined twice, once in `loadLana` again in `lana.js`. The file load only happens on call to log, but the iife inside `lana.js` may not have finished yet.